src/utils/useAtomsDevtools.ts

```diff
@@ -9,7 +9,7 @@ import { SnapshotOptions, useAtomsSnapshot } from './useAtomsSnapshot';
 import { useGotoAtomsSnapshot } from './useGotoAtomsSnapshot';
 
 const atomToPrintable = (atom: AnyAtom) =>
-  atom.debugLabel ? `${atom}:${atom.debugLabel}` : `${atom}`;
+  atom.debugLabel && !String(atom).includes(':') ? `${atom}:${atom.debugLabel}` : `${atom}`;
```

Comment:
alternatively should this just be?
```js
const atomToPrintable = (atom: AnyAtom) => String(atom)
```

The caveat is that atomToPrintable will not include debugLabel in prod.

Reply:
Another idea
```ts
const atomToPrintable = (atom: AnyAtom) =>
  atom.debugLabel && import.meta.env?.MODE === 'production' ? `${atom}:${atom.debugLabel}` : `${atom}`;
```

Reply:
Actually, current is probably the best because it is forwards and backwards compatible
```ts
const atomToPrintable = (atom: AnyAtom) =>
   atom.debugLabel && !String(atom).includes(':') ? `${atom}:${atom.debugLabel}` : `${atom}`;
```

Reply:
I'm personally fine with:
```ts
const atomToPrintable = (atom: AnyAtom) => `${atom}`;
```

But, to be extra safe, something like:
```ts
const atomToPrintable = (atom: AnyAtom) => {
  let str = `${atom}`;
  if (atom.debugLabel && !str.endsWith(":" + atom.debugLabel)) {
    str += ":" + atom.debugLabel;
  }
  return str;
};
```

Reply:
Ah, I see. We only need to check `includes(":")`.